install.sh

```diff
@@ -994,11 +994,24 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
+# show different instructions on adding Homebrew to path based on if the user has already done so
```

Comment:
Should I change the wording of this comment?

Reply:
Since the intent is (well, should be) clear from the code, I'm not sure it serves a useful purpose to begin with.

Reply:
Shall I remove it then?

Reply:
I would, but I'm not a maintainer. :grinning: